include/nbl/ext/MitsubaLoader/CElementEmitter.h

```diff
@@ -251,6 +255,12 @@ class CElementEmitter : public IElement
 					animation = anim;
 					return true;
 					break;*/
+				case IElement::Type::EMISSION_PROFILE: {
+					if (type == Type::AREA) {
+						area.emissionProfile = static_cast<CElementEmissionProfile*>(_child);
+					}
+					return true;
```

Comment:
we return `true` on success, and `false` on failure, this means that if you don't expect (i.e. not AREA, POINT or SPOT) to have an IES profile, you should return false... so move the `return true` into the if-clause

Reply:
Also above you have a `toWorld`  handling path, which didn't use to handle AREA, now you need to make it handle AREA since thats what orients the emission profile.

Right now you'll get a parsing error from 
```xml
<shape type="sphere">
      <emitter type="area">
           <rgb name="radiance" value="1"/>
           <emissionprofile type="measured">
              <string name="filename" value="test.ies"/>
              <float name="normalizeEnergy" value="12.56636"/>
           </emissionprofile>
           <transform name="toWorld">
              <lookat origin="1, 1, 1" target="1, 2, 1" up="0, 0, 1"/>
          </transform>
    </emitter>
</shape>
```